### PR TITLE
Sign Out  button, common UI style

### PR DESCRIPTION
**Pull request by 92nikolaev**

### CODE REVIEW

**Review by Fevor7 — APPROVED** [review]

File: backend/interview_portal/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+charset=utf-8
+end_of_line=crlf
+insert_final_newline=false
+indent_style=space
+indent_size=4
+
+
+[*.json]
+indent_style=space
+indent_size=2
```
Comment (Yauheni-Verameichyk) [inline backend/interview_portal/.editorconfig:11]: Should be in git ignore

File: frontend/src/app/users/user/user.component.html [hunk 2]
```diff
@@ -2,9 +2,14 @@
     <td>
         {{user.name}}</td>
     <td>{{user.surname}}</td>
-    <td >
-        <span *ngFor="let role of user.roles">{{transformStyle(role)}}<br></span>
```
Comment (Yauheni-Verameichyk) [inline frontend/src/app/users/user/user.component.html:6]: Transform style method move into model as getter

File: frontend/src/app/shared/validator/validator-messge/validator-message.directive.spec.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { ValidatorMessageDirective } from '/validator-messge/validator-message.directive.ts';
```
Comment (aliaksandr-aliaksandrau) [inline frontend/src/app/shared/validator/validator-messge/validator-message.directive.spec.ts]: Please rename folder to '/validator-message' from '/validator-messge'

File: frontend/src/app/users/user-list/user-list.component.html [hunk 4]
```diff
@@ -1,14 +1,13 @@
 <br>
 <table class="user-table table table-hover table-bordered">
-  <thead class="thead-light">
+  <thead style="background-color: #e3f2fd;">
```
Comment (aliaksandr-aliaksandrau) [inline frontend/src/app/users/user-list/user-list.component.html]: Please try to use 'class' instead of 'style'

**Review by Yauheni-Verameichyk — COMMENTED** [review]
Comments below

File: frontend/src/app/disciplines/disciplines-list/disciplines-list.component.ts [hunk 5]
```diff
@@ -29,6 +29,7 @@ export class DisciplinesListComponent implements OnInit, OnDestroy {
       .takeUntil(this.destroy)
       .subscribe((disciplines) => {
         this.disciplinesList = disciplines;
+        console.log(disciplines);
```
Comment (Yauheni-Verameichyk) [inline frontend/src/app/disciplines/disciplines-list/disciplines-list.component.ts]: Please delete console.log()

**Pull request merged**